docs/src/pages/components/time-picker.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import {
+  SectionTitle,
+  Title,
+  TimePicker,
+  ListItem,
+  List,
+  Link
+} from "@nulogy/components";
+import Highlight from "react-highlight";
+import {
+  Layout,
+  Intro,
+  IntroText,
+  DocSection,
+  PropsTable
+} from "../../components";
+import radioProps from "../../shared/radioProps";
```

Comment:
this doesn't seem right. and i actually just noticed it's on the datepicker docs too. (https://nulogy.design/components/date-picker/)

Reply:
oops totally forgot to add the props tables for both :/  will do that now